CodeStarterCamp_Week2/main.swift

```diff
@@ -8,5 +8,51 @@
 
 import Foundation
 
-print("Hello, World!")
+func pickLottoNumbers() -> Set<Int> {
+    var randomNumber = 0
+    var lottoNumbers = Set<Int>()
+    while lottoNumbers.count < 6 {
+        randomNumber = Int.random(in: 1...45)
+        lottoNumbers.insert(randomNumber)
+    }
+    return lottoNumbers
+}
+
+func compareSameNumbers(lottoNumbers: Set<Int>, myLottoNumbers: Array<Int>) -> Array<Int> {
+    var sameNumbers = Array<Int>()
+    for myNumber in myLottoNumbers {
+        for lottoNumber in lottoNumbers {
+            if myNumber == lottoNumber {
+                sameNumbers.append(myNumber)
+            }
+        }
+    }
+    return sameNumbers
+}
+
+func compareSameNumbers2(lottoNumbers: Set<Int>, myLottoNumbers: Array<Int>) -> Set<Int> {
+    let setMyLottoNumbers = Set(myLottoNumbers)
+    let sameNumbers: Set<Int> = setMyLottoNumbers.intersection(lottoNumbers)
+    return sameNumbers
+}
+
+
+func printSameNumbers(lottoNumbers: Set<Int>, myLottoNumbers: Array<Int>) {
+    let sameNumbers = compareSameNumbers(lottoNumbers: lottoNumbers, myLottoNumbers: myLottoNumbers)
+    if sameNumbers.count > 0 {
+        var StringSameNumbers = [String]()
+        for num in sameNumbers {
+            StringSameNumbers.append(String(num))
+        }
+        StringSameNumbers.sort()
+        print("축하합니다! 겹치는 번호는 \(StringSameNumbers.joined(separator: ", ")) 입니다!")
+    } else {
+        print("아쉽지만 겹치는 번호가 없습니다.")
+    }
+}
```

Comment:
함수 안의 코드 컨벤션이 대체적으로 개행이 없다보니 가독성이 안 좋은 거 같아요!
```suggestion
func printSameNumbers(lottoNumbers: Set<Int>, myLottoNumbers: Array<Int>) {
    let sameNumbers = compareSameNumbers(lottoNumbers: lottoNumbers, myLottoNumbers: myLottoNumbers)
    
    if sameNumbers.count > 0 {
        var StringSameNumbers = [String]()
        
        for num in sameNumbers {
            StringSameNumbers.append(String(num))
        }
        
        StringSameNumbers.sort()
        print("축하합니다! 겹치는 번호는 \(StringSameNumbers.joined(separator: ", ")) 입니다!")
    } else {
        print("아쉽지만 겹치는 번호가 없습니다.")
    }
}
```
제 코드 컨벤션 스타일이긴 하지만, 가독성을 위해서 위의 함수들도 적절한 개행을 해주셨으면 합니다!

Reply:
@jsa0224
아예 같은 요소를 확인하는 함수가 있었군요!🥲 contains를 사용해서 for문 한번만 사용하도록 수정해 보고 개행도 넣었습니다.